api/agent/agent.go

```diff
@@ -895,16 +895,24 @@ func (a *agent) runHot(ctx context.Context, caller slotCaller, call *call, tok R
 	go func() {
 		defer cancel() // also close if we get an agent shutdown / idle timeout
 
+		// We record init wait for two cases below: I. when it succeeds, II. when it times out,
+		// other cases are unusual early exits (eg. agent shutdown, eviction, errors, etc)
+		// and we assume that those cases are already tracked separately and should not pollute
```

Comment:
OOC, is this assumption true?

Reply:
updated with tags to avoid this assumption.